extension/src/util/object.ts

```diff
@@ -26,3 +26,7 @@ export const createTypedAccumulator = <T extends string>(
   }
   return acc
 }
+
+type BaseType = string | number | boolean | Object | undefined | null
```

Comment:
I needed these guys to fix the auto-generated types, so I copied them from the webview module

Reply:
I think we can put them into a central contract file and import them into the webview instead of duplicating them.